db/seeds/alot_of_jobs/job_seed.js

```diff
@@ -0,0 +1,758 @@
+    })
+
+    .then(function () {
+      return knex('jobs').insert({
+        id: '138',
+        accountId: 'ALOT-123',
+        title: 'Title-138',
+        employer: 'Employer-138',
+        created_at: moment().subtract(week).add(day),
+        updated_at: moment().subtract(week).add(day),
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-12',
+        rating: '3',
+        status: 'applied',
+        status_sort_index: 1,
+      });
+    })
+
+    .then(function () {
+      return knex('jobs').insert({
+        id: '139',
+        accountId: 'ALOT-123',
+        title: 'Title-139',
+        employer: 'Employer-139',
+        created_at: moment().subtract(month).add(day),
+        updated_at: moment().subtract(month).add(day),
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-12',
+        rating: '3',
+        status: 'interview',
+        status_sort_index: 2,
+      });
+    })
+
+    .then(function () {
+      return knex('jobs').insert({
+        id: '140',
+        accountId: 'ALOT-123',
+        title: 'Title-140',
+        employer: 'Employer-140',
+        created_at: moment().subtract(month).add(day),
+        updated_at: moment().subtract(month).add(day),
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-12',
+        rating: '3',
+        status: 'interview',
+        status_sort_index: 2,
+      });
+    })
+
+    .then(function () {
+      return knex('jobs').insert({
+        id: '141',
+        accountId: 'ALOT-123',
+        title: 'Title-141',
+        employer: 'Employer-141',
+        created_at: moment().subtract(month).add(day),
+        updated_at: moment().subtract(month).add(day),
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-12',
+        rating: '3',
+        status: 'interview',
+        status_sort_index: 2,
+      });
+    })
+
+    .then(function () {
+      return knex('jobs').insert({
+        id: '142',
+        accountId: 'ALOT-123',
+        title: 'Title-142',
+        employer: 'Employer-142',
+        created_at: moment().subtract(fortnight).add(day),
+        updated_at: moment().subtract(fortnight).add(day),
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-12',
+        rating: '3',
+        status: 'interview',
+        status_sort_index: 2,
+      });
+    })
+
+    .then(function () {
+      return knex('jobs').insert({
+        id: '143',
+        accountId: 'ALOT-123',
+        title: 'Title-143',
+        employer: 'Employer-143',
+        created_at: moment().subtract(week).add(day),
+        updated_at: moment().subtract(week).add(day),
+        sourceType: 'online',
+        sourceUrl: 'http://www.stuff.com',
+        deadline: '2016-07-12',
+        rating: '3',
+        status: 'interview',
+        status_sort_index: 2,
```

Comment:
we are getting more and more job definitions in our codebase. What will happen if the model will change? It may be painful to synchronise all of this job definitions.